rotkehlchen/tests/utils/exchanges.py

```diff
@@ -355,12 +444,18 @@ def create_test_coinbase(
 def create_test_binance(
         database: DBHandler,
         msg_aggregator: MessagesAggregator,
+        location: Location = Location.BINANCE,
 ) -> Binance:
+    if location == Location.BINANCE:
+        uri = BINANCE_BASE_URL
+    elif location == Location.BINANCE_US:
+        uri = BINANCE_US_BASE_URL
```

Comment:
should add an "else BOOM - AssertionError" here. But in another PR. Since this is just test code.